### PR TITLE
make strategy more conservative

### PR DESCRIPTION
**Pull request by hal909**

### CODE REVIEW

**Review by barrutko — COMMENTED** [review]
Maybe we also want to add some sort of a setter, so we won't need more upgrades in the future?

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
I agree with Bartek, let's make this an owner-settable variable since we're already interested in changing it once.

**Review by yuchenlintt — APPROVED** [review]

File: contracts/truefi2/strategies/CurveYearnStrategy.sol [hunk 1]
```diff
@@ -28,8 +28,6 @@ contract CurveYearnStrategy is UpgradeableClaimable, ITrueStrategy {
 
     // Number of tokens in Curve yPool
     uint8 public constant N_TOKENS = 4;
-    // Max slippage during the swap
-    uint256 public constant MAX_PRICE_SLIPPAGE = 200; // 2%
```
Comment (barrutko) [inline contracts/truefi2/strategies/CurveYearnStrategy.sol:32]: Are we sure that we can remove constant from the memory? @vanruch?

Comment (barrutko) [reply contracts/truefi2/strategies/CurveYearnStrategy.sol:32]: Nevermind.

Comment (yuchenlintt) [reply contracts/truefi2/strategies/CurveYearnStrategy.sol:32]: My understanding is that constants do not affect contract storage. See the Solidity [docs](https://docs.soliditylang.org/en/latest/contracts.html#constants) for reference:

> It is also possible to define constant variables at the file level.
>
> The compiler does not reserve a storage slot for these variables, and every occurrence is replaced by the respective value.

**Comment by barrutko** [comment]
Not sure if such conservative requirements will ever pass, though.

**Comment by yuchenlintt** [comment]
> Not sure if such conservative requirements will ever pass, though.

Yeah I agree. Knowing how frequently our integration tests already fail, I don't know if setting the maxPriceSlippage to 0.75% would ever succeed.

Regardless, this PR should give us the option to adjust down _or up_, as needed.

**Comment by vanruch** [comment]
Integration tests passed on first try, LGTM

**Pull request merged**